recipes/django-axes/LICENSE

```diff
@@ -0,0 +1,23 @@
+The MIT License
```

Comment:
The file `LICENSE` is already included in the tarball, so there's no need to put a separate copy in the recipe here.

Reply:
fixed this